data/vibe/data/json.d

```diff
@@ -181,19 +181,20 @@ struct Json {
 	}
 
 	/// New JSON value of Type.Undefined
-	static @property Json undefined() { return Json(); }
+	static @property Json undefined() nothrow { return Json(); }
 
 	/// New JSON value of Type.Object
-	static @property Json emptyObject() { return Json(cast(Json[string])null); }
+	static @property Json emptyObject() nothrow { return Json(cast(Json[string])null); }
 
 	/// New JSON value of Type.Array
-	static @property Json emptyArray() { return Json(cast(Json[])null); }
+	static @property Json emptyArray() nothrow { return Json(cast(Json[])null); }
 
 	version(JsonLineNumbers) int line;
 
 	/**
 		Constructor for a JSON object.
 	*/
+    nothrow {
```

Comment:
I'd repeat this for every constructor for now to be in line with the rest. In the current form I think it may also break the generated documentation.